src/Oci8/Oci8Connection.php

```diff
@@ -236,10 +236,48 @@ public function executeFunction($sql, array $bindings = [], $returnType = PDO::P
     }
 
     /**
+     * Execute a PL/SQL Procedure and return its result.
+     * Usage: DB::executeProcedure($procedureName, $bindings)
+     *
+     * $bindings looks like:
+     *         $bindings = [
+     *                  'p_userid'  => [
+     *                      'value' => $id,
+     *                      'type'  => PDO::PARAM_INT,
+     *                  ]
+     *         ],
+     *
+     * @param $procedureName
+     * @param $bindings
+     * @return array
+     */
+    private function executeProcedure($procedureName, $bindings)
+    {
+        $command = sprintf('begin %s(:%s, :cursor); end;', $procedureName, implode(', :', array_keys($bindings)));
+
+        $stmt = $this->getPdo()->prepare($command);
+
+        foreach ($bindings as $bindingName => $bindingOption) {
+            $stmt->bindParam(':' . $bindingName, $bindingOption['value'], $bindingOption['type']);
+        }
+
+        $cursor = null;
+
+        $stmt->bindParam(':cursor', $cursor, PDO::PARAM_STMT);
```

Comment:
What if we don't want to return a cursor on the procedure? Maybe it would be more cool if we can set the return type like on the function method?